pom.xml

```diff
@@ -108,15 +108,61 @@
 
     <dependencyManagement>
         <dependencies>
-            <!-- project dependencies -->
+            <!-- project dependencies - runtime and compile -->
+            <dependency>
+                <groupId>io.kroxylicious</groupId>
+                <artifactId>kroxylicious</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.kroxylicious</groupId>
+                <artifactId>kroxylicious-api</artifactId>
+                <version>${kroxyliciousApi.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.kroxylicious</groupId>
+                <artifactId>kroxylicious-filter-api</artifactId>
+                <version>${kroxyliciousApi.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.kroxylicious</groupId>
+                <artifactId>kroxylicious-multitenant</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.kroxylicious</groupId>
+                <artifactId>kroxylicious-schema-validation</artifactId>
+                <version>${project.version}</version>
+            </dependency>
```

Comment:
I'm fine with this for now but I think the right answer, at least in part is https://github.com/kroxylicious/kroxylicious/issues/400. There are also a couple of chicken and egg questions with that however.